src/main/java/frc/robot/RobotContainer.java

```diff
@@ -162,19 +229,48 @@ public void configureObjects() {
     private void configureButtonBindings() {
       Joystick leftJoystick = new Joystick(0);
       Joystick rightJoystick = new Joystick(1);
+      Joystick altJoystick = new Joystick(2);
 
       
-      JoystickButton balanceChargingStationButton = new JoystickButton(rightJoystick, 1);
-      JoystickButton turtleButton = new JoystickButton(rightJoystick, 3);
-      JoystickButton fieldCentricButton = new JoystickButton(leftJoystick, 2);
+
+      JoystickButton balanceChargingStationButton = new JoystickButton(leftJoystick, 1);
+      JoystickButton turtleButton = new JoystickButton(rightJoystick, 1);
+      JoystickButton fieldCentricButton = new JoystickButton(rightJoystick, 2);
+
+      JoystickButton pivotUpButton = new JoystickButton(altJoystick, 4);
+      JoystickButton pivotDownButton = new JoystickButton(altJoystick, 1);
+
+      JoystickButton telescopeOutButton = new JoystickButton(altJoystick, 2);
+      JoystickButton telescopeInButton = new JoystickButton(altJoystick, 3);
+
+      JoystickButton elevatorUpButton = new JoystickButton(altJoystick, 6);
+      JoystickButton elevatorDownButton = new JoystickButton(altJoystick, 5);
+
+      JoystickButton toggleGrabberButton = new JoystickButton(altJoystick, 9);
+
+      Trigger pivotToTopPegButton = new Trigger(() -> altJoystick.getPOV() == 0);
```

Comment:
Nice custom trigger use

Reply:
This was the reason we ignored the D-Pad last year 😆 glad we figured it out this year!